### DISCUSSION TITLE
Selling an assembled Klimerko device

### DISCUSSION THREAD

**Discussion by mayeaux** [DesconBelgrade/Klimerko#44] [General]
Hi there. I'd like to participate with my own Klimerko device but I don't really have the capacity (really time) to take on assembling my own device. Given that a proprietary air pollution sensor can cost hundreds of dollars, would someone be able to assemble and sell me a working Klimerko device that I can hook up to the cloud myself? Thanks!

**Comment by PavelVoronovBeo** [comment]
Plus one with the same issue )

**Reply by EcoPanter** [reply]
If you want Klimerko device you can write on svetkakavzelis@gmail.com and someone will contact you.
